File: mobile-client/android/Contract/Message/Thread/MessageItemPayload.kt
package com.smartresponsor.mobile.client.contract.message.thread

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class MessageItemPayload(
    val messageId: String,
    val threadId: String,
    val body: String,
    val senderId: String,
    val sentAtIso8601: String,
)

File: client/ios/Contract/Message/Thread/MessageItemPayload.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
public struct MessageItemPayload: Sendable, Codable {
    public let messageId: String
    public let threadId: String
    public let body: String
    public let senderId: String
    public let sentAtIso8601: String
}
